DEBUG some7 T7 client-test7 MDC-TEST7 [main] SocketServerTestCase - Message 1
DEBUG some7 T7 client-test7 MDC-TEST7 [main] root - Message 2
 INFO some7 T7 client-test7 MDC-TEST7 [main] SocketServerTestCase - Message 3
 WARN some7 T7 client-test7 MDC-TEST7 [main] SocketServerTestCase - Message 4
LETHAL some7 T7 client-test7 MDC-TEST7 [main] SocketServerTestCase - Message 5
DEBUG some7 T7 client-test7 MDC-TEST7 [main] SocketServerTestCase - Message 6
java.lang.Exception: Just testing
	at org.apache.log4j.net.SocketServerTestCase.common(SocketServerTestCase.java:XXX)
	at org.apache.log4j.net.SocketServerTestCase.test7(SocketServerTestCase.java:XXX)
	at junit.framework.TestCase.runTest(TestCase.java:XXX)
	at junit.framework.TestCase.runBare(TestCase.java:XXX)
	at junit.framework.TestResult$1.protect(TestResult.java:XXX)
	at junit.framework.TestResult.runProtected(TestResult.java:XXX)
	at junit.framework.TestResult.run(TestResult.java:XXX)
	at junit.framework.TestCase.run(TestCase.java:XXX)
	at junit.framework.TestSuite.runTest(TestSuite.java:XXX)
	at junit.framework.TestSuite.run(TestSuite.java:XXX)
ERROR some7 T7 client-test7 MDC-TEST7 [main] root - Message 7
java.lang.Exception: Just testing
	at org.apache.log4j.net.SocketServerTestCase.common(SocketServerTestCase.java:XXX)
	at org.apache.log4j.net.SocketServerTestCase.test7(SocketServerTestCase.java:XXX)
	at junit.framework.TestCase.runTest(TestCase.java:XXX)
	at junit.framework.TestCase.runBare(TestCase.java:XXX)
	at junit.framework.TestResult$1.protect(TestResult.java:XXX)
	at junit.framework.TestResult.runProtected(TestResult.java:XXX)
	at junit.framework.TestResult.run(TestResult.java:XXX)
	at junit.framework.TestCase.run(TestCase.java:XXX)
	at junit.framework.TestSuite.runTest(TestSuite.java:XXX)
	at junit.framework.TestSuite.run(TestSuite.java:XXX)
